### ISSUE TITLE
Grammar on file truncated message

### ISSUE THREAD

**Issue by philiprbrenan** [geany/geany#1003]
![zzz](https://cloud.githubusercontent.com/assets/7548378/14763072/00f010ee-0983-11e6-961b-666bb410fc5e.png)

Please consider using the phrase:  

"The file might have been truncated by this action"

which uses the subjunctive to imply uncertainty,  the present past to indicate an action which occurred in the past but which has consequences now in the present and the passive voice to indicate uncertainty as to who performed the action.

The message as it stands indicates that some unknown agency has permission to truncate the file  in the immediate future if they wish to - which is not the case.

Thanks!

**Comment by codebrainz** [comment]
I don't think that minor tweak goes far enough to improve this error. Look at the first two sentences:

> Error saving file.

> Error opening file ...

I assume the "opening" part has something to do with atomic rename after save operation, but the user shouldn't need to understand such implementation details in order to understand the error message. Also, the last line is needlessly scary and non-descriptive. Perhaps it could something more like:

> Please correct the error and try again or save the document in another location before closing it to avoid losing work.
